types/unit/timer/timerspec.pp

```diff
@@ -0,0 +1,4 @@
+# @summary Timer specification for systemd timers
+# @see https://www.freedesktop.org/software/systemd/man/systemd.timer.html
+#
+type Systemd::Unit::Timer::TimerSpec = Variant[Integer[0],String,Array[Variant[Integer[0],String]]]
```

Comment:
```suggestion
type Systemd::Timespan = Variant[Integer[0],String,Array[Variant[Integer[0],String]]]
```
From `man systemd.timer` and `man systemd.time` these values seem to be called `time span`s

Looking around these "Time Spans" are also used in say `Restart` of [Service],  

Not suggesting all of those should be adjusted to this now but it make sense I think to add `Timespan` more generic than the time directory. Makes them less silly long as well seems a good idea.

Reply:
updated.